app/scripts/views/tooltip.js

```diff
@@ -27,14 +28,6 @@ define([
       this.dismissible  = options.dismissible || false;
       this.extraClassNames = options.extraClassNames || '';
 
-      // Event codes for the keyboard keys.
-      this.DOWN_ARROW_KEY_CODE = 40;
-      this.ENTER_KEY_CODE = 13;
-      this.LEFT_ARROW_KEY_CODE = 37;
-      this.RIGHT_ARROW_KEY_CODE = 39;
-      this.TAB_KEY_CODE = 9;
-      this.UP_ARROW_KEY_CODE = 38;
-
```

Comment:
related to https://github.com/mozilla/fxa-content-server/pull/3220/files#r43058886